frontend/views/search/_results.html.erb

```diff
@@ -4,21 +4,25 @@
 
 <%
   # Start of modifications
-  # See https://github.com/archivesspace/archivesspace/blob/master/frontend/app/helpers/search_helper.rb
+  # Customize accession columns only in search results, when creating linked entities
   model = "accession"
-  add_pref_columns(model)
-  # Add content_description column at end
-  prop = 'content_description'
-  enum_locale_key = "#{model}_#{prop}"
-  add_column(
-    I18n.t("#{model}.#{prop}"),
-    { locale_key: enum_locale_key, model: model, sortable: false },
-    proc { |record|
-      v = record[prop] || ASUtils.json_parse(record['json'])[prop]
-      I18n.t("enumerations.#{enum_locale_key}.#{v}", :default => v.to_s) unless v.nil?
-    }
-  )
-  add_actions_column if !params[:linker] || params[:linker] === 'false'
+  if !@search_data.nil? && @search_data.get_type == model
```

Comment:
`@search_data.get_type` returns `multi` if there are multiple resource types returned by the search, or the type name if there's just one. Checking for "accession" here means that Clements will get the custom column they want whenever only accessions are in the results, but search data with multiple types will get the default columns with a useful "Resource Type" column. I think this is an improvement over the previous version, where you'd always get "Content Description" even for other entity types, and you couldn't easily distinguish between an Accession and a Resource, say, in the results.

Reply:
https://github.com/archivesspace/archivesspace/blob/305b9a3a11579f28472e0a4948803c2a7a8e959d/frontend/app/models/search_result_data.rb#L215-L234